update_database_scripts/unpackDatabaseCommandLine.R

```diff
@@ -444,16 +530,23 @@ if (is_ebi_reachable()) {
         tidyr::extract(range, into = c("lowerCI", "upperCI"),regex = "(\\d+.\\d+)-(\\d+.\\d+)") %>%
         add_column(citation = citation) %>%
         add_column(studyID = studyID, .after = "citation") %>%
-        add_column(sex = getSexesFromDescriptions(master_associations[["pvalue_description"]])) %>%
+        add_column(sex = getSexesFromDescriptions(master_associations[["pvalue_description"]], master_associations[["beta_description"]])) %>%
+        add_column(betaValue = getBeta(master_associations[["beta_number"]], master_associations[["beta_direction"]]), .after = "upperCI") %>%
+        add_column(ogValueTypes = getOGValueTypes(master_associations[["or_per_copy_number"]], master_associations[["beta_number"]]), .after = "betaValue") %>%
         mutate(pvalue_description = tolower(pvalue_description))
-      # filter out SNPs on the X or Y chromosome
-      studyData <- filter(studyData, !startsWith(hg38, "X")&!startsWith(hg38, "Y"))
+      # filter out SNPs on the X or Y chromosome (keeping NAs)
```

Comment:
Is this line ensuring that when we have a valid rsid, but the chrome:pos is NA we keep it?

Reply:
Yes